### PR TITLE
Fixed default cloud test

### PR DESCRIPTION
**Pull request by AsafMah**
### Fixed
* Fixed test not working in dev envs
* Fixed build w/ black

### CODE REVIEW

File: azure-kusto-data/azure/kusto/data/_cloud_settings.py [hunk 1]
```diff
@@ -18,43 +19,24 @@
 DEFAULT_PUBLIC_LOGIN_URL = "https://login.microsoftonline.com"
 DEFAULT_REDIRECT_URI = "https://microsoft/kustoclient"
 DEFAULT_KUSTO_SERVICE_RESOURCE_ID = "https://kusto.kusto.windows.net"
+DEFAULT_DEV_KUSTO_SERVICE_RESOURCE_ID = "https://kusto.dev.kusto.windows.net"
```
Comment (yihezkel) [inline azure-kusto-data/azure/kusto/data/_cloud_settings.py:22]: Can we not get these from the Metadata endpoint?

**Review by yihezkel — APPROVED** [review]
See comment, please. Approved.

File: azure-kusto-data/azure/kusto/data/_cloud_settings.py (hunk 1 above)
Comment (AsafMah) [reply azure-kusto-data/azure/kusto/data/_cloud_settings.py:22]: The whole point of these is to have a default when we are unable to get it from metadata (originally it was for clusters that didn't deploy the feature yet)

Comment (AsafMah) [reply azure-kusto-data/azure/kusto/data/_cloud_settings.py:22]: Resolved offline

**Pull request merged**